### PR TITLE
[appveyor] Fix command line

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

Discovery the joy of cmd programming...

### CODE REVIEW

File: phpunit [hunk 1]
```diff
@@ -51,11 +54,13 @@ if ($phpIniMatrix) {
 
     $phpDir = dirname(`where.exe php`);
 
-    $newCmd = '(SET X=0';
+    $newCmd = 'cmd /v:on /d /c "(SET X=0';
```
Comment (stof) [inline phpunit]: are you sure that windows flags are case insensitive ? they are uppercase in the Process component

File: phpunit [hunk 2]
```diff
@@ -51,11 +54,13 @@ if ($phpIniMatrix) {
...
     foreach ($phpIniMatrix as $iniFile) {
-        $newCmd .= " & copy /Y $phpDir\\$iniFile $phpDir\\php.ini & echo. & echo Running tests with $iniFile: & $cmd & (if %%errorlevel%% NEQ 0 SET X=1)";
+        $newCmd .= " & copy /Y $phpDir\\$iniFile $phpDir\\php.ini & echo. & echo Running tests with $iniFile: & ($cmd || SET X=1)";
```
Comment (stof) [inline phpunit]: shouldn't it be ``&&`` rather than ``&`` btw ?

File: phpunit (hunk 1 above)
Comment (nicolas-grekas) [inline phpunit]: yes, I tested on my windows vm, this works fine

File: phpunit (hunk 2 above)
Comment (nicolas-grekas) [inline phpunit]: `&` is the bash equivalent of `;`, which is what we want: a failure in a matrix line should not prevent the second line from running

Comment (nicolas-grekas) [inline phpunit]: (and I assume that copy and echo will never fail yes :) )

**Pull request merged**